README.md

```diff
@@ -1,288 +1,171 @@
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 [Set up a Hello-react-Frontend] <a name="about-project"></a>
 
-**[your_project__name]** is a...
+> This is a Set up a Set up a Hello-react-Frontend project .
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://ruby-lang.org/">Ruby</a></li>
+  </ul>
+</details>
 
 <details>
-  <summary>Client</summary>
+<summary>Database</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
   </ul>
 </details>
 
 <details>
-  <summary>Server</summary>
+  <summary>Linters</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://rubocop.org/">Ruby</a></li>
   </ul>
 </details>
 
 <details>
-<summary>Database</summary>
+  <summary>Test</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://rspec.info/">Ruby</a></li>
   </ul>
 </details>
 
 <!-- Features -->
 
-### Key Features <a name="key-features"></a>
-
-> Describe between 1-3 key features of the application.
+### Live Demo
+Here is our [Project Demonstration](#) 
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+Here is our [Project path](https://github.com/frnega/hello-react-front-end)
 
-<!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
+### Key Features <a name="key-features"></a>
 
-- [Live Demo Link](https://google.com)
+- **random generation of message**
+- **use webpack**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
-### Prerequisites
-
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
-
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
 
 ```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  cd PATH
+  git clone https://github.com/frnega/hello-react-front-end
 ```
---->
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
 ```sh
-  cd my-project
-  gem install
+  cd hello-react-front-end
+  npm install
 ```
---->
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
-  rails server
+  npm start
 ```
---->
 
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
-
+rspec
 ```sh
-  bin/rails test test/models/article_test.rb
+  
 ```
```

Comment:
**[OPTIONAL]**

- [ ] It would very nice if you add `npm test` here instead of `rspec` because `rspec` is a command for rails not for react :thinking: